dinput_hook/renderer_utils.h

```diff
@@ -0,0 +1,51 @@
+#pragma once
+
+#include "types.h"
+#include <glad/glad.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+void renderer_setProjectionMatrix(rdMatrix44 *projectionMatrix);
```

Comment:
whats the reason those functions are exported and extern "C"? should not really be needed.

Reply:
Scaffolding not needed anymore 👍 . I used these functions to emulate `glMatrixMode` and they arent needed anymore. I'll remove them

Reply:
`renderer_setAlphaMask` is used in `std3D.c: std3D_Startup` and `n64_shader.cpp: set_render_mode` for example